Zend/zend_builtin_functions.c

```diff
@@ -1851,7 +1851,7 @@ static int copy_function_name(zval *zv, int num_args, va_list args, zend_hash_ke
 		char *disable_functions = INI_STR("disable_functions");
 
 		if ((*exclude_disabled == 1) && (disable_functions != NULL)) {
-			if (strstr(disable_functions, func->common.function_name->val) == NULL) {
+			if (func->internal_function.handler != ZEND_FN(display_disabled_function)) {
```

Comment:
You can now also drop the INI_STR fetch above (which is not terribly efficient...)

Reply:
… and simplify further. Thanks again!